lesson24/src/js/main.js

```diff
@@ -75,6 +75,7 @@ submitButton.addEventListener("click", (e) => {
 
 const options = {
     root: document.querySelector('[data-id="modal-inner"]'),
+    threshold: 1
 };
```

Comment:
I added the threshold that Sae-san taught me. 
I was able to achieve the movement I wanted. Thank you!

https://user-images.githubusercontent.com/99260576/199499190-7d6b9218-8522-4543-8d02-d0d7788d127e.mov